ts/components/SelectCalendarModal.tsx

```diff
@@ -53,6 +55,7 @@ type Props = {
   onCancel: () => void;
   onCalendarSelected: (calendar: Calendar) => void;
   header?: React.ReactNode;
+  defaultCalendar?: Calendar | undefined;
```

Comment:
it's either `defaultCalendar?: Calendar` or `defaultCalendar: Calendar | undefined`

Reply:
Maybe caused by rush, fixed. Thx.